.travis.yml

```diff
@@ -1,14 +1,14 @@
+sudo: required
```

Comment:
Yeah, I agree it's same problem.

However I'm reluctant to hardcode the version for this one, for MRI
rubies I follow closely the different releases, and don't mind so much
maintaining patch versions manually (I'd prefer not to of course).

But for jruby I don't follow at all, don't use it, and don't even have
it on my dev machine to be honest, I rely on travis only.

I'll ask for a third opinion on this one.